src/ducks/account/LoanDetailsPage.jsx

```diff
@@ -1,11 +1,260 @@
+      <CompositeRow
+        className={cx(styles.LoanDetailsRow__inner)}
+        primaryText={title}
+        secondaryText={caption}
+        right={right}
+        {...rest}
+      />
+    </div>
+  )
+})
+
+const Section = props => {
+  const { title, children } = props
+
+  return <BaseSection title={title}>{children}</BaseSection>
+}
+
+const hasInfos = (account, paths) => {
+  for (const path of paths) {
+    if (get(account, path)) {
+      return true
+    }
+  }
+
+  return false
+}
+
+const KeyInfosSection = translate()(props => {
+  const { account, t } = props
+
+  const shouldRender = hasInfos(account, [
+    'loan.usedAmount',
+    'balance',
+    'loan.rate'
+  ])
+
+  if (!shouldRender) {
+    return null
+  }
+
+  return (
+    <Section title={t('Informations clés')}>
+      <Row
+        type="amount"
+        title={t('Montant emprunté')}
+        value={get(account, 'loan.usedAmount')}
+      />
+      <Row
+        type="amount"
+        title={t('Capital restant dû')}
+        value={-get(account, 'balance')}
+      />
+      <Row
+        type="rate"
+        title={t("Taux d'intérêt")}
+        value={get(account, 'loan.rate')}
+      />
+    </Section>
+  )
+})
+
+const PaymentsSection = translate()(props => {
+  const { account, t, f } = props
+
+  const shouldRender = hasInfos(account, [
+    'loan.lastPaymentDate',
+    'loan.nextPaymentDate'
+  ])
+
+  if (!shouldRender) {
+    return null
+  }
+
+  return (
+    <Section title={t('Paiements')}>
+      <Row
+        type="amount"
+        title={t('Dernier paiement effecté')}
+        value={get(account, 'loan.lastPaymentAmount')}
+        caption={'le ' + f(get(account, 'loan.lastPaymentDate'), DATE_FORMAT)}
+      />
+      <Row
+        type="amount"
+        title={t('Prochain paiement prévu')}
+        value={get(account, 'loan.nextPaymentAmount')}
+        caption={
+          t('le ') + f(get(account, 'loan.nextPaymentDate'), DATE_FORMAT)
+        }
+      />
+    </Section>
+  )
+})
+
+const CharacteristicsSection = translate()(props => {
+  const { account, t } = props
+
+  const shouldRender = hasInfos(account, [
+    'loan.subscriptionDate',
+    'loan.maturityDate',
+    'loan.nbPaymentsLeft',
+    'loan.nbPaymentsDone'
+  ])
+
+  if (!shouldRender) {
+    return null
+  }
+
+  return (
+    <Section title={t('Caractéristiques du prêt')}>
+      <Row
+        type="date"
+        title={t('Date de souscription')}
+        value={get(account, 'loan.subscriptionDate')}
+      />
+      <Row
+        type="date"
+        title={t('Date de remboursement')}
+        value={get(account, 'loan.maturityDate')}
+      />
+      <Row
+        type="nb-payments"
+        title={t("Nb d'échéances encore prévues")}
+        value={get(account, 'loan.nbPaymentsLeft')}
+      />
+      <Row
+        type="nb-payments"
+        title={t("Nb d'échéances payées")}
+        value={get(account, 'loan.nbPaymentsDone')}
+      />
```

Comment:
Untranslated, I guess you know but I'd be more confortable with a big fat comment "// TODO i18n" ;)